### ISSUE TITLE
clarify documentation regarding supported unicode ranges

### ISSUE THREAD

**Issue by paregorios** [isawnyu/pleiades-gazetteer#295]
Was: "Egyptological alef is not supported in name strings"

A user writes:

> it looks like the Egyptological aleph character is not supported. I used a "3" in its place, which seems to be the standard replacement when that character is not supported. If that's incorrect, please let me know!

**Comment by paregorios** [comment]
Cannot reproduce. Seeking more info from user.

**Comment by steko** [comment]
For the record, the character under discussion is `ꜣ` (lower case) or `Ꜣ` (upper case).

**Comment by paregorios** [comment]
@steko yep, thanks. I was able to enter and view it myself, so I'm trying to find out what prompted this report.

![screen shot 2017-03-28 at 2 13 28 pm](https://cloud.githubusercontent.com/assets/263285/24425124/9297dc48-13c8-11e7-8a8f-3bf786aacb5a.png)

**Comment by paregorios** [comment]
User says:
> I was going by the list of supported Unicode blocks-- it's in Latin Extended-D, but the Pleiades documentation lists only A through C as supported.... It's this page:
https://pleiades.stoa.org/help/name-romanization-guide. It does specify that those are the character sets for the "Romanized name" field (though the character in question does turn up in romanizations).

**Comment by paregorios** [comment]
Pleiades Romanizations are strictly that: Romanizations (hence no Egyptological Alef); however, it's perfectly legal in the "attested" field. Some further clarification to this extent is probably warranted in the docs.
